docker/Dockerfile.nvidia

```diff
@@ -129,59 +118,5 @@ RUN if [ "x$MPI_VER" = "x4" ]; then \
                $HPCSDK_HOME/comm_libs/mpi ; \
     fi;
 
-# Install pip dependencies and devito as a pip package
```

Comment:
Leftover comment. This now only builds the compiler devito part is in the final dependent one.

Reply:
Oups wrong response. Basically now there is two steps
- The base images (nvidia, nvidia-clang) that do not contain anything but the compilers and env var
- The actual docker one `Dockerfile` that just adds devito and all these extra on top of the compilers.
So all of that is now only in `Dockerfile` which is the one used to build the actual devito images while this only build the SDK and related stuff. 

This way we can have really quick build since devito is small compared to the time it takes to build the sdk/llvm.